tests/files/README.md

```diff
@@ -0,0 +1,10 @@
+# Test files #
+
+This folder contains a collection of test files to cover different use cases of the lzma-rs library.
+
+This README describes files that are not self-explanatory in this folder.
+
+## bad-random-data
```

Comment:
Could you make the name a bit more specific than "bad random data"? Something like `range-coder-edge-case` would be clearer I think. Also, in the description below, can you document how it was created?

Reply:
Good idea. Updated now :)